src/Grpc/test/InteropTests/Helpers/WebsiteProcess.cs

```diff
@@ -78,7 +78,17 @@ private void Process_OutputDataReceived(object sender, DataReceivedEventArgs e)
 
         public void Dispose()
         {
-            File.WriteAllText(_serverLogPath ?? Path.Combine(Directory.GetCurrentDirectory(), "InteropServer.log"), _consoleOut.ToString());
+            if (!string.IsNullOrEmpty(_serverLogPath))
```

Comment:
I'll revert this commit if https://github.com/dotnet/aspnetcore/pull/20005#pullrequestreview-378116740 goes in.

Reply:
@HaoK so eventually when #20005 goes in, it'll be correct to log directly to the current directory right?

Reply:
No, you should write to artifacts/logs, might need to create that directory

Reply:
so CurrentDirectory/artifacts/logs?

Reply:
https://github.com/dotnet/aspnetcore/pull/20005 is finally merged, so maybe try one more rebase/build to see if the logs are working if you want